src/oop/user_defined_classes/hard/q2/QuestionText.md

```diff
@@ -0,0 +1,30 @@
+# oop-user_defined_classes-hard-q2
+
+## Notes
+
+- Vary both of the attributes for this question and the class name
+- Also vary the method name
+
+## Question Text
+
+Dishwashers are convenient, but it can be easy to lose track of how much clean cutlery you have. Write a Java class 
+called `DishWasher` to track the contents of the dishwasher. See the description below for the requirements.
+
+### DishWasher Class:
+
```

Comment:
For all of these also add what visibility modifiers they need for example:
An int called forks which is not visible from other classes.
meaning that it should be private.